docs/developers.md

```diff
@@ -1,7 +1,140 @@
 This document is a work in progress.
 
+If it lacks some information that you needed, consider adding it via a PR.
```

Comment:
This document is amazing! I would remove the first line and add "create an issue" as an alternative for reporting. People may have feedback, but not the time to apply it.

I would suggest changing the 3 lines to:

```md
_Wasm Language Runtimes is evolving constantly so this is a live document. If you find any issue, typo or missing information, feel free to open an issue or create a PR directly_
```